misc/semver-checker/modules/semver-checker-core/src/main/java/io/ballerina/semver/checker/comparator/DumbNodeComparator.java

```diff
@@ -37,9 +37,9 @@
  */
 public class DumbNodeComparator<T extends Node> implements Comparator {
 
-    protected final T newNode;
-    protected final T oldNode;
-    protected DiffKind nodeKindName;
+    private final T newNode;
+    private final T oldNode;
+    private final DiffKind nodeKindName;
```

Comment:
Wouldn't it be better to have `nodeKind` instead of `nodeKindName`, as other places it's been named as `nodeKind`?

Reply:
Yeah that makes sense. Updated with [c17b9fd](https://github.com/ballerina-platform/ballerina-lang/pull/37482/commits/c17b9fd7282ec2ec3691300c2dfc70bc84c8cdd9)